### DISCUSSION TITLE
自带商店安装软件 导致系统卡顿

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4333] [Troubleshooting | 故障排查]
用自带的应用商店安装软件的时候，风扇明显转的异常的快，关键我就是装了个小软件而已，又不是什么大软件。而且，不仅会导致系统卡顿，连我的鼠标移动都变得很卡顿（我用的是无线鼠标）。

另外，应用商店里面的一款名叫“智力游戏”的游戏有严重BUG,在五子棋游戏里面，如果我不按照系统给出的答案走棋的话，电脑也会变得严重卡顿并且风扇狂转，游戏也不会给出任何提示一直卡住。
https://bbs.deepin.org/phone/zh/post/256349

**Comment by BLumia** [comment]
机器配置不太行？装软件本身都会导致卡的话怀疑机器配置是不是有点低了...给点详细信息？
